internal/pkg/template/workload.go

```diff
@@ -481,6 +481,7 @@ type WorkloadOpts struct {
 	AppName      string
 	EnvName      string
 	WorkloadName string
+	Manifest     string // Serialized manifest used to deploy the workload.
```

Comment:
Maybe `SerializedManifest` or `RawManifest` so that we don't need to comment?

Reply:
Done! renamed also environment template's .Manifest field in `env.go` to `.SerializedManifest`